### PR TITLE
Modify documentation on the Injecting Extra Context.

### PR DESCRIPTION
**Pull request by karantan**
ref: #863

@hackebrot here it is. Thoughts?

### CODE REVIEW

File: docs/advanced/injecting_content.rst [hunk 1]
```diff
@@ -8,11 +8,20 @@ You can specify an `extra_context` dictionary that will override values from `co
...
-This is a sample Python script that dynamically injects a timestamp value
-as a project is generated::
+If you have ``cookiecutter.json`` that has the following keys::
+
+    {
+        "timestamp": {{ cookiecutter.timestamp }}
+    }
+
+
+This Python script will dynamically injects a timestamp value as a project is
```
Comment (michaeljoseph) [inline docs/advanced/injecting_content.rst]: > will dynamically **inject**

File: docs/advanced/injecting_content.rst [hunk 2]
```diff
@@ -8,11 +8,20 @@ You can specify an `extra_context` dictionary that will override values from `co
...
+
 Example: Injecting a Timestamp
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This is a sample Python script that dynamically injects a timestamp value
-as a project is generated::
+If you have ``cookiecutter.json`` that has the following keys::
+
+    {
+        "timestamp": {{ cookiecutter.timestamp }}
```
Comment (michaeljoseph) [inline docs/advanced/injecting_content.rst]: `{{cookiecutter.timestamp}}` probably needs to be surrounded with quotes?

File: docs/advanced/injecting_content.rst (hunk 1 above)
Comment (michaeljoseph) [inline docs/advanced/injecting_content.rst]: > as **the** project is generated ?

**Comment by michaeljoseph** [comment]
:bell: @karantan

**Comment by karantan** [comment]
@michaeljoseph I'll fix it at the end of the week (I just created reminder for this). I've been swamped with other stuff.

**Comment by karantan** [comment]
@michaeljoseph PR fixed. Sorry it took so long.

**Comment by theodesp** [comment]
I think this PR is good to merge! @michaeljoseph

**Comment by michaeljoseph** [comment]
:+1: Thanks @karantan!

**Pull request merged**